modules/openapi-generator/src/main/resources/rust-server/Cargo.mustache

```diff
@@ -5,7 +5,9 @@ authors = [{{#infoEmail}}"{{{.}}}"{{/infoEmail}}]
 {{#appDescription}}
 description = "{{{.}}}"
 {{/appDescription}}
-license = "Unlicense"
+{{#licenseInfo}}
+license = "{{.}}"
+{{/licenseInfo}}
```

Comment:
if licenseInfo is not provided, can we default it to `license = "Unlicense ` ?

Reply:
Sure.  Can I also add a comment explaining that the license can be set using the `info.license` field in the openapi spec?

Reply:
Im ok with that.